### DISCUSSION TITLE
Mail Consume - IMAP Folders at GMail

### DISCUSSION THREAD

**Discussion by Dark-Gizmoa** [jonaswinkler/paperless-ng#1268] [Support]
Hi,

i'm a little bit lost at the configuration for Mail Consume, specifically at the IMAP Folder configuration for my GMail Account.

Lets say, that all consume mails get the Tag / Folder "Paperless".
I tried and set the IMAP Folder to paperless, but the mails are ignored / not found.

Can anybody help me with the correct configuration for gmail / userdefined imap tags?

**Comment by sti0** [comment]
Maybe you find an answer yourself but I use a label / folder `Paperless` and in the email rule I entered the same `Paperless`. Maybe it needs to be case sensitive.

**Comment by crzdmniac** [comment]
> Hi,
> 
> i'm a little bit lost at the configuration for Mail Consume, specifically at the IMAP Folder configuration for my GMail Account.
> 
> Lets say, that all consume mails get the Tag / Folder "Paperless". I tried and set the IMAP Folder to paperless, but the mails are ignored / not found.
> 
> Can anybody help me with the correct configuration for gmail / userdefined imap tags?

If you haven't figured this out, or someone finds this thread later, Google doesn't really follow standards with their IMAP implementation. Their "tags" are really folders in IMAP world, and I don't believe tags do anything (I haven't been able to figure anything out at least). My workaround was to set a rule in Gmail to put the email with PDF attachments in what essentially becomes a temporary folder, I have Paperless look for that "folder" (I know Gmail's verbage is tag), and my rule in paperless is to "Move to folder" and I have it move it back to the inbox, which removes that tag and leaves it only with Inbox.
